packages/transformer/src/EntityUnifier.ts

```diff
@@ -18,6 +18,7 @@ import {
 import {
   ConcreteEntity,
   ConcreteEntityProps,
+  // eslint-disable-next-line @typescript-eslint/no-redeclare
   Element,
```

Comment:
It was confusing to me why all the Element imports flag this, especially since it doesn't seem to be a problem in itwinjs-core.

Reply:
I think it means we're contributing the typescript lib compiler option to include the browser types

Reply:
ah interesting, well if youre fine with me individually ignoring each line feel free to resolve.

Or do we even need the browser types?